src/llmcompressor/args/dataset_arguments.py

```diff
@@ -117,6 +117,16 @@ class DatasetArguments(CustomDatasetArguments):
         default=512,
         metadata={"help": "Number of samples to use for one-shot calibration"},
     )
+    calibrate_moe_context: bool = field(
```

Comment:
Shouldn't this basically always be on? Is there ever a case where a user shouldn't use this?

Reply:
Hm, but there's no conflict between `prepare_for_calibration` and `calibrate_moe_context`, right? I think it'd also look a little confusing to calibrate an MoE model, but explicitly call
```python3
prepare_for_calibration(my_moe_model)
oneshot(my_moe_model, calibrate_moe_context=False)
```

Reply:
Would it be better to just remove it and always run with `calibrate_moe_context=True`?

We require `prepare_for_calibration` to be explicitly applied, the idea was to be the same with the calibration context.